### PR TITLE
Remove notion of credit from explainers

### PR DESCRIPTION
**Pull request by apasel422**
***
<a href="https://pr-preview.s3.amazonaws.com/apasel422/conversion-measurement-api/pull/172.html" title="Last updated on Jul 12, 2021, 1:02 PM UTC (0c3b08f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/conversion-measurement-api/172/51de375...apasel422:0c3b08f.html" title="Last updated on Jul 12, 2021, 1:02 PM UTC (0c3b08f)">Diff</a>

### CODE REVIEW

File: event_attribution_reporting_clicks.md [hunk 1]
```diff
@@ -250,13 +248,7 @@ delete the scheduled report with the lowest priority and schedule the new report
...
 ### Multiple sources for the same trigger (Multi-touch)
 
-If multiple sources were clicked and associated with a single attribution trigger, send reports for all of them. 
-
-To provide additional utility, the browser can choose to provide additional annotations to each of these reports, attributing credits for the triggering redirect to them individually. Attribution models allow for more sophisticated, accurate measurement.
-
-The default attribution model will be last-click attribution, giving the last-clicked source for a given trigger redirect all of the credit.
-
-To remain flexible, the browser sends an `credit` of value 0 to 100 for all reports associated with a single trigger. This represents the percent of attribution a source received. The sum of credits across a set of reports for one trigger should equal 100.
+If multiple sources were clicked and associated with a single attribution trigger, send reports for the one with the highest priority.
```
Comment (johnivdel) [inline event_attribution_reporting_clicks.md:251]: source priority isn't a concept in this explainer.

Should we upstream attributionsourceprioirty in this as well?

If not this should read, send a report for the most recent source if we wanted to be self-consistent in the explainer. But adding in priority makes the most sense imo, as it is already in the spec.

Comment (apasel422) [reply event_attribution_reporting_clicks.md:254]: Done.

File: event_attribution_reporting_clicks.md [hunk 2]
```diff
@@ -103,18 +103,21 @@ Attribution Source Declaration
 An attribution source is an anchor tag with special attributes:
 
 `<a attributiondestination="[eTLD+1]" attributionsourceeventid="[64-bit unsigned integer]"
-attributionexpiry=[unsigned long long] attributionreportto="[origin]">`
+attributionexpiry=[unsigned long long] attributionreportto="[origin]"
+attributionsourcepriority="[64-bit unsigned integer"]>`
```
Comment (johnivdel) [inline event_attribution_reporting_clicks.md:107]: Priority is signed https://wicg.github.io/conversion-measurement-api/#dom-htmlanchorelement-attributionsourcepriority

File: event_attribution_reporting_clicks.md [hunk 3]
```diff
@@ -250,13 +251,7 @@ delete the scheduled report with the lowest priority and schedule the new report
 
 ### Multiple sources for the same trigger (Multi-touch)
 
-If multiple sources were clicked and associated with a single attribution trigger, send reports for all of them. 
-
-To provide additional utility, the browser can choose to provide additional annotations to each of these reports, attributing credits for the triggering redirect to them individually. Attribution models allow for more sophisticated, accurate measurement.
-
-The default attribution model will be last-click attribution, giving the last-clicked source for a given trigger redirect all of the credit.
```
Comment (johnivdel) [inline event_attribution_reporting_clicks.md:257]: Let's note that by default if no priority is specified, the browser performs last-touch

File: event_attribution_reporting_views.md [hunk 4]
```diff
@@ -90,9 +90,7 @@ See the [privacy considerations section](#privacy-considerations) for the ration
 
 ### Controlling which attribution source to trigger
 
-In the Click Through Explainer, at triggering time, the browser looks up all matching sources in storage. A [credit](https://github.com/WICG/conversion-measurement-api#multiple-sources-for-the-same-trigger-multi-touch) of 100 is assigned to the most recent matching source in storage, and a credit of 0 is assigned to the rest. Reports are sent for each of these. We propose changing this for both click sources and event sources.
-
-To provide additional utility in prioritizing the distribution of credit between different classes of sources, the browser can associate a priority with each attribution source. This priority will be defined in a new attribute on attribution source tags, `attributionsourcepriority`, which holds a 64 bit integer.
+The browser associates a priority with each attribution source. This priority will be defined in a new attribute on attribution source tags, `attributionsourcepriority`, which holds a 64 bit integer.
```
Comment (johnivdel) [inline event_attribution_reporting_views.md:93]: nit: Now that this is in the clicks explainer, this isn't a new attribute.

File: event_attribution_reporting_views.md [hunk 5]
```diff
@@ -91,9 +91,7 @@ See the [privacy considerations section](#privacy-considerations) for the ration
 
 ### Controlling which attribution source to trigger
 
-In the Click Through Explainer, at triggering time, the browser looks up all matching sources in storage. A [credit](https://github.com/WICG/conversion-measurement-api#multiple-sources-for-the-same-trigger-multi-touch) of 100 is assigned to the most recent matching source in storage, and a credit of 0 is assigned to the rest. Reports are sent for each of these. We propose changing this for both click sources and event sources.
-
-To provide additional utility in prioritizing the distribution of credit between different classes of sources, the browser can associate a priority with each attribution source. This priority will be defined in a new attribute on attribution source tags, `attributionsourcepriority`, which holds a 64 bit integer.
+The browser associates a priority with each attribution source. This priority will be defined in an attribute on attribution source tags, `attributionsourcepriority`, which holds a 64-bit signed integer.
```
Comment (johnivdel) [inline event_attribution_reporting_views.md:94]: On second thought wdyt about removing this section for the most part. The paragraph below is restating the information in the clicks explainer. We can maybe just say:

```suggestion
Attribution sources with different `source_types` can be prioritized among one another using the `[attributionsourcepriority](https://github.com/WICG/conversion-measurement-api/blob/main/event_attribution_reporting_clicks.md#multiple-sources-for-the-same-trigger-multi-touch)` attribute proposed in the clicks explainer.
```

**Review by johnivdel — APPROVED** [review]

**Pull request merged**